training/train.py

```diff
@@ -206,7 +206,7 @@ def train(model, criterion, optimizer, lr_scheduler, train_dataset, val_dataset,
                 rename_latest_to_best(latest_file_paths)
 
 
-def main(yaml_path, domain):
+def main(yaml_path, model_path, domain):
```

Comment:
could you change model_path into "pretrained_model_path" to clarify that the model_path means the pre-trained model's path.

Reply:
OK. I understood.